src/overlays/actors/ovl_En_Ending_Hero6/z_en_ending_hero6.c

```diff
@@ -9,9 +9,40 @@ void EnEndingHero6_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnEndingHero6_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnEndingHero6_Draw(Actor* thisx, GlobalContext* globalCtx);
 
+void func_80C23D60(EnEndingHero6* this, s32 npcIndex);
+void func_80C23DDC(EnEndingHero6* this);
 void func_80C23E18(EnEndingHero6* this, GlobalContext* globalCtx);
 
-#if 0
+extern FlexSkeletonHeader D_0600B0CC; // object_dt
+extern FlexSkeletonHeader D_06007908; // object_bai
+extern FlexSkeletonHeader D_06007150; // object_toryo
+extern FlexSkeletonHeader D_0600D640; // object_sdn
+extern FlexSkeletonHeader D_0600A850; // object_daiku
+
+extern AnimationHeader D_06000BE0; // object_dt
+extern AnimationHeader D_060011C0; // object_bai
+extern AnimationHeader D_06000E50; // object_toryo
+extern AnimationHeader D_06002A84; // object_sdn
+extern AnimationHeader D_06002FA0; // object_daiku
+
+// object_daiku
+extern Gfx D_060070C0[];
+extern Gfx D_06006FB0[];
+extern Gfx D_06006E80[];
+extern Gfx D_06006D70[];
+extern Gfx D_0600A390_dl[];
+
+// object_dt
+extern TexturePtr D_06007350;
+extern TexturePtr D_06009590;
+extern TexturePtr D_06009F90;
+extern TexturePtr D_0600A790;
+extern TexturePtr D_0600AB90;
+
+extern TexturePtr D_06007750;
+extern TexturePtr D_0600A390_tex;
```

Comment:
```suggestion
extern TexturePtr D_0600A390;
```
Same idea as my previous comment. You might not even need a comment since it is prefaced with TexturePtr